app/models/post.rb

```diff
@@ -103,8 +103,7 @@ def notify
     # Reply from user back to the system
     elsif self.kind == "reply" && self.user_id != self.topic.user_id && self.topic.private?
       I18n.with_locale(self.email_locale) do
-        # NOTE New ticket is misnamed, it should be new-reply
-        PostMailer.new_post(self).deliver_later
+        PostMailer.new_post(id).deliver_later
```

Comment:
Thats a good catch because #436 was trying to make the jobs thread safe by passing in IDs to the mailer instead of the object.  I don't think this matters with `suckerpunch` but it does with `sidekiq`.  It still worked correctly passing `self` but passing `id` or `self.id` is much better.